WordPress/gutenberg
# [Widgets Editor] Fix insertion point in widget areas

## Description

Fix #25708.
Fixes https://github.com/WordPress/gutenberg/issues/25764.

Insert blocks at the bottom of the widget area via the inserter.

Also fixes a regression in #25708 where the inserter would insert blocks to the first widget area no matter which widget area is selected.

## How has this been tested?



1. Go to Widgets Editor
2. Select the second widget area
3. Insert a block via the inserter
4. The block should appear at the bottom of the second widget area, and scroll down to it.

## Screenshots 
![Kapture 2020-09-30 at 13 56 38](https://user-images.githubusercontent.com/7753001/94648283-d3ad7b80-0324-11eb-8005-821296e93c0b.gif)

## Types of changes




Bug fix

## Checklist:
